BSA Supplies Cargo Crate now comes with a guide paper

Peliex opened the pull request:
## About The Pull Request
Wrote up a small step by step guide to assembling the bsa, in case someone isn't really sure since it's a bit different from building a normal machine.
![image](https://github.com/tgstation/tgstation/assets/20830349/f1d7a823-e681-4de0-b305-a90f95ae5222)


## Why It's Good For The Game
Makes it easier to build for people that don't know how, I feel like that's probably a good thing.

## Changelog
:cl:Senefi
qol: Bluespace Artillery Cargo Crate includes a guide paper for how to assemble it.
/:cl:

Peliex committed:
adds guide paper for building the bluespace artillery

Peliex committed:
lowercase ts

EuclidianGeometer commented:
This document should also mention it needs to be activated by swiping a card in two head of department offices.

BlueMemesauce commented on code/modules/paperwork/paper_premade.dm:
```diff
@@ -31,6 +31,45 @@
+	* Bluespace Artillery Controls
+	* 26 cable pieces
+	* 20 iron
+	* 20 bluespace crystals (or synthetic)
```
```suggestion
	* 20 bluespace crystals (or artificial)
```

BlueMemesauce commented on code/modules/paperwork/paper_premade.dm:
```diff
@@ -31,6 +31,45 @@
+	* 5 femto servos
+	* 1 glass sheet
+	<h1>2. Find suitable location</h1>
+	The BSA has a steep power requirement and functions best when it is on its own APC.  Keep other items in the room to a minimum. Ensure the BSA is placed somewhere where it will not shoot at another part of the station.  Make sure the room isn't easily accessible by someone who shouldn't be there.
```
```suggestion
	The BSA has a steep power requirement and functions best when it is on its own APC; keep other machines in the room to a minimum. Ensure the BSA is placed at the edge of the station, so it will not shoot through another part of the station.  Make sure the room isn't easily accessible by someone who shouldn't be there.
```

BlueMemesauce commented on code/modules/paperwork/paper_premade.dm:
```diff
@@ -31,6 +31,45 @@
+	8. Install the fusor in the center machine frame.
+	9. Install the generator in the back machine frame.
+	10. Examine the three machines and see what else they need, you can use a screwdriver to finish the assembly when parts are installed.
+	11. Construct a computer terminal in front of the fusor, it must be directly south of it, wrench it to the ground and add the Controls board.
```
I don't believe this is true, the console can be built in other locations. Which is often better as it can be in a secure location like the bridge.

Peliex replied:
I'll give it a shot, I've only ever actually built the console in the same room as the BSA.  Also thank you for the corrections! I'll fix the grammatical errors and add in the information about the console.

Peliex replied:
The console can be built in the same room as the BSA, they actually must exist in the same area but that is slightly technical and not necessary info for construction.  Trying to move the console to another room (even the bridge) makes it unable to detect the BSA assembly.

Peliex replied:
If the CE wants to put it on the bridge they'll have to do some work themselves to make that happen, and they (hopefully) already know how to build a BSA.

E231kei500bandai reviewed (commented):
some suggestions

Peliex commented:
> This document should also mention it needs to be activated by swiping a card in two head of department offices. 

That's a good thing to note, thank you, I'll add the fact that it needs authorization to fire.

Peliex committed:
Fixed grammatical errors, thanks memesauce!

playing bg3 i'll open up the game later to check if the console has to be next to the bsa or not.

Peliex committed:
Merge branch 'master' into objective-instructions

Peliex committed:
Removed false info about BSA console, included authorization requirement

Peliex committed:
Merge branch 'master' into objective-instructions
